backend/media/covers/dejan/repo/master/README.md

```diff
@@ -0,0 +1 @@
+Generated by GitUKS v1.0.0 on 2021-06-08 21:29:11.612803
```

Comment:
Treba voditi racuna da se izbaci posle ovo

Reply:
Greškom otišlo i to, sad sam sredio.